tools/workspace-plugin/package.json

```diff
@@ -7,6 +7,6 @@
   "scripts": {
     "test": "nx run @fluentui/workspace-plugin:test",
     "type-check": "nx run @fluentui/workspace-plugin:type-check",
-    "lint": "nx run @fluentui/workspace-plugin:lint-old"
```

Comment:
this was not properly executing eslint with our specific configuration, moved to RAW binary. will be redone once we migrate to nx on pipelines